tests/js/spec/views/performance/transactionSummary/transactionEvents/eventsTable.spec.jsx

```diff
@@ -0,0 +1,235 @@
+import {mountWithTheme} from 'sentry-test/enzyme';
```

Comment:
Is there a possibility for this test file to be written in typescript? i.e. `eventsTable.spec.tsx`?

Reply:
Updated!